terraform/cloudwatch.tf.json.template.py

```diff
@@ -173,6 +195,81 @@ def prod_qualified_resource_name(name: str) -> str:
                     }
                     for lambda_ in config.lambda_names()
                 ),
+                {
+                    'aws_cloudwatch_metric_alarm': {
+                        **{
+                            f'internet_{direction}': {
+                                'alarm_name': config.qualified_resource_name(f'internet_{direction}'),
+                                'comparison_operator': 'GreaterThanThreshold',
+                                'threshold': threshold,
+                                'evaluation_periods': 1,
+                                'datapoints_to_alarm': 1,
+                                'treat_missing_data': 'notBreaching',
+                                'metric_query': [
+                                    {
+                                        'id': f'internet_{direction}',
+                                        'label': f'Internet {direction} bytes/h',
+                                        'expression': ' + '.join(f'm{zone}' for zone in range(vpc.num_zones)),
+                                        'return_data': True,
+                                    },
+                                    *(
+                                        {
+                                            'id': f'm{zone}',
+                                            'metric': {
+                                                'dimensions': {
+                                                    'NatGatewayId': f'${{data.aws_nat_gateway.gitlab_{zone}.id}}'
+                                                },
+                                                'namespace': 'AWS/NATGateway',
+                                                'metric_name': metric_name,
+                                                'period': 1 * 60 * 60,
+                                                'stat': 'Sum',
+                                            }
+                                        }
+                                        for zone in range(vpc.num_zones)
+                                    )
+                                ],
+                                'alarm_actions': ['${data.aws_sns_topic.monitoring.arn}'],
+                                'ok_actions': ['${data.aws_sns_topic.monitoring.arn}'],
+                            }
+                            for direction, metric_name, threshold in [
+                                ('ingress', 'BytesInFromDestination', 50 * 1024 * 1024 * 1024),
+                                ('egress', 'BytesOutToDestination', 10 * 1024 * 1024 * 1024)
```

Comment:
How where this values determined?

Reply:
> How where this values determined?

The threshold values were specified by Hannes in a [comment](https://github.com/DataBiosphere/azul/issues/5282#issuecomment-1585127316) on the source ticket.

Reply:
I missed that, thank you.